### PR TITLE
Bugfix/picture close automatically if click green dot on top screen

### PR DESCRIPTION
**Pull request by lhtrung307**
Fix picture closes automatically if you open a green dot that is on the top of the screen

### CODE REVIEW

File: resources/js/views/Locations/Countries.vue [hunk 1]
```diff
@@ -61,13 +61,13 @@
                     <div class="columns">
                         <div class="column is-half is-offset-3">
                             <div class="columns">
-                                <div class="column is-one-third">
-                                    <h1 class="subtitle is-5" style="color: black; text-align: center;">
```
Comment (xlcrr) [inline resources/js/views/Locations/Countries.vue:65]: Why are we removing the black here?

File: resources/js/views/Locations/Countries.vue [hunk 2]
```diff
@@ -61,13 +61,13 @@
...
+                                <div class="column has-text-centered">
+                                    <h1 class="subtitle is-6">
                                         <strong style="color:black;">
                                             {{ $t('location.maps10') }}
                                         </strong>
                                     </h1>
-                                    <h1 class="title is-2" style="text-align: center;">
+                                    <h1 class="title is-2">
```
Comment (xlcrr) [inline resources/js/views/Locations/Countries.vue:70]: Why are we removing `center` here?

**Review by xlcrr — CHANGES_REQUESTED** [review]
A lot of similarities here between this and #45  

What does this PR fix exactly ?

File: resources/js/views/Locations/Countries.vue (hunk 1 above)
Comment (lhtrung307) [reply resources/js/views/Locations/Countries.vue:65]: We already have `style="color:black;"` on the `strong` tag.

File: resources/js/views/Locations/Countries.vue (hunk 2 above)
Comment (lhtrung307) [reply resources/js/views/Locations/Countries.vue:70]: We already have `has-text-centered` on the `div` tag.

**Comment by lhtrung307** [comment]
Like I said in slack, this for `Fix picture closes automatically if you open a green dot that is on the top of the screen`. And you should merge it after PR#45. I want to use the eslint so I created this branch base on that.

**Review by xlcrr — APPROVED** [review]

**Pull request merged**